daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/EngineTest.scala

```diff
@@ -507,6 +507,7 @@ class EngineTest
           lookupPackage,
         )
       isReplayedBy(tx, rtx) shouldBe Right(())
+      txMeta.nodeSeeds shouldBe newMeta.nodeSeeds
```

Comment:
:+1:

Reply:
Maybe we could have one more involved test that actually uses rollback node.

Reply:
A test for `submit` also would be nice. This really applies to everything as soon as a `try` is involved.